_data/simple-icons.json

```diff
@@ -830,6 +845,11 @@
             "hex": "CC2929",
             "source": "https://support.humblebundle.com/hc/en-us/articles/202742060-Bundle-Logos"
         },
+        {
+            "title": "IconJar",
+            "hex": "109CE8",
```

Comment:
The color is defined [in their press kit](https://geticonjar.com/press-kit/) to be `#16A5F3`